code/modules/mining/equipment/mineral_scanner.dm

```diff
@@ -61,6 +62,7 @@
 		current_cooldown = world.time + cooldown
 		var/turf/t = get_turf(src)
 		mineral_scan_pulse(t, range)
+		playsound(src, 'sound/effects/ping.ogg', 20)
```

Comment:
Please add a toggle with alt click or something, this gets super annoying when it's every two seconds for an hour and a half.

Reply:
Done.